module-system/module-implementations/examples/sov-value-setter/Cargo.toml

```diff
@@ -27,6 +27,7 @@ serde_json = { workspace = true, optional = true }
 thiserror = { workspace = true }
 borsh = { workspace = true, features = ["rc"] }
 jsonrpsee = { workspace = true, features = ["macros", "client-core", "server"], optional = true }
+clap = { workspace = true }
```

Comment:
We should include clap only when compiled with native feature (for all the modules)

Reply:
yup, this was a mistake. i'll enable it only for native. this is also the reason the prover is failing in CI - there are a few crates i included while testing but didn't restrict them to native

Reply:
Fixed